pkg/controller/metrics/diverters/MetricsSenderToKafka.go

```diff
@@ -0,0 +1,89 @@
+package diverters
+
+import (
+	"context"
+	"fmt"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/segmentio/kafka-go"
+	"strconv"
+)
+
+type KafkaMetricsSender struct {
+	Queue chan model.MetricSpec
```

Comment:
Can you explain why use struct here, I prefer to change it to a pointer.

Reply:
changed to 
```Queue    chan *model.MetricSpec```